src/main/app/src/app/shared/edit/edit-datum-groep/edit-datum-groep.component.less

```diff
@@ -64,24 +64,23 @@
 }
 
 .play-control {
-  &:hover {
+  &:hover, &.hervat {
```

Comment:
Als de zaak opgeschort is geweest dan moet je niet de gedisabelde pauze-knop laten zien maar het play-icon. Pas als je er overheen hoovert moet de gdisablede pauze-knop zichtbaar worden.

Reply:
Dit heb ik juist zo afgestemd met Roy, dat de gedisabelde knop getoond wordt zonder dat die hover nodig is. Lijkt mij vanuit UX-perspectief ook logischer, dat je niet als gebruiker op een knop wil drukken en pas op het moment dat je er dichtbij zit dat je er achter komt dat dat niet kan/mag.

Reply:
Besproken via teams - blijkbaar was de eerste feedback van Roy niet correct. Oude hover-gedrag weer hersteld in nieuwe commit.